docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -87,6 +87,14 @@ For help with specific yugabyted commands, run 'yugabyted [ command ] -h'. For e
 $ ./bin/yugabyted start -h
 ```
 
+## Base directory
+
+By default, yugabyted uses the `/home/user/var` directory to store data, configurations, and logs.
+
+You can change the base directory when starting a cluster using the `--base_dir` flag. If you change the base directory, you _must_ specify the base directory using the `--base-dir` flag when running subsequent commands on the cluster.
+
```

Comment:
Better we add an example here for using custom `--base-dir`

If you override the base_dir when starting the yugabyted node like below -
```
./bin/yugabyted start --base_dir /home/user/node1
```
Any subsequent yugabyted commands should include the `--base_dir`

```
./bin/yugabyted status  --base_dir  /home/user/node1
```